### DISCUSSION TITLE
GitHub banned my account without any comms and support ghosts me

### DISCUSSION THREAD

**Discussion by wrclark** [community/community#88590] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am creating this post on behalf of my friend.

Hi,

My ticket number is: 2528922

I contacted you last weekend but still haven't received any further contact from you.

My account, github.com/mishka, got banned without any warning. I didn't have a chance to back up any of my repos.

I don't think people wake up and think about getting banned from GitHub just like that.

Here it says: "Reinstatement: Where a user can agree to abide by our Acceptable Use Policies moving forward and has made the changes necessary to address the violation(s), we may choose to reinstate their account or content depending on the circumstances and severity of the initial violation."

https://docs.github.com/en/site-policy/acceptable-use-policies/github-appeal-and-reinstatement

I don't know what I've done, but I am keen on collaborating on the issue and removing/making changes accordingly to get my account back.

I use it for LinkedIn and many job applications and would like to have it back. This is a significant manner.

Please get in touch with me about this as soon as you can because I feel ignored now.

Kind Regards,

**Comment by ghost** [comment]
anyone?

**Reply by schlagmichdoch** [reply]
@mishka @wrclark I'm currently facing the same problem. I'm glad to see that https://github.com/mishka is reachable again. How were you able to resolve your issue? :)

**Reply by stil-blue** [reply]
@mishka @wrclark I'm currently facing the same problem on my main account https://github.com/schlagmichdoch. I'm glad to see that https://github.com/mishka is reachable again.

How were you able to resolve your issue? :)

**Reply by mishka** [reply]
I don't know how, but my account was hacked, and it was hosting virus repos. I didn't realize it because I don't check my github that often. They deactivated my account to protect it due to suspicious activity.

The support replied within two weeks and reactivated my account after clearing all the login tokens.

It takes a while until they respond, but you can only wait for now.

They can help you as well, but it will take some time.
![GFhBen3bkAAZPU0](https://github.com/community/community/assets/25265212/0e85288e-f3d9-4607-bee2-395c613c5b19)

**Reply by stil-blue** [reply]
Thanks for the quick reply! I guess it’s 2 weeks silence then🥹
Too bad they don’t even give any explanation at all at why they shadow hide your account..
I’m pretty sure my account was not hacked, but I had indeed received an email about suspicious activity.

We will see.
Thanks again and take care! :)

**Comment by DipayanC2012** [comment]
What happened next? I am facing similar issue

**Comment by DipayanC2012** [comment]
Similar issue, my ticket no is #2578342 . I have had no response in the last 3 weeks. Someone please help.

**Comment by nathaniyelll** [comment]
Hi,

My main account github.com/nathaniyell with ticket number 2613482, has been suspended for over a week now. I have filled the Appeal and Reinstatement form but there's been no response from github till date. Is there any hope of the reinstatement process going through? I just got a mail:
"Hello @Nathaniyell,

We recently noticed that your GitHub account had a suspicious login. Out of an abundance of caution, we made the decision to force a password reset for the account associated with this email address."

The password reset link they provided isn't working as intended. Please I need help!

**Comment by K-Naveenkumar1** [comment]
response2

**Comment by wooter007** [comment]
Similar issue, my ticket no is #2729225. 

Another question, how to check the progress? When I resubmitted the appeal, I was told that the relevant issues had been updated. But I didn't receive the email.
![image](https://github.com/community/community/assets/130435106/ce822c4e-b9e8-4909-95ec-dc06aa02d497)
